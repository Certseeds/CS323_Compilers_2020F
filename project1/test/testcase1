123
456
21415
21313
1.114514
dsada_dsad12
2.35
0.0002E+2
