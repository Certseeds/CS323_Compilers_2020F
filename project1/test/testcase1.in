int test_spl(){
    int i = 0,j = 1;
    float i = 1;
}