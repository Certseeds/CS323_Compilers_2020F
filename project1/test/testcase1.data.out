TYPE int
ID test_spl
LP
RP
LC
TYPE int
ID i
ASSIGN
INT 0
COMMA
ID j
ASSIGN
INT 1
SEMI
TYPE float
ID i
ASSIGN
INT 1
SEMI
RC
